pkg/build/builder/util/dockerfile/instructions_test.go

```diff
@@ -65,7 +65,7 @@ func TestKeyValueInstructions(t *testing.T) {
 			in: []KeyValue{
 				{"☃", "'\" \\ / \b \f \n \r \t \x00"},
 			},
-			want: `"☃"="'\" \\ / \u0008 \u000c \n \r \t \u0000"`,
+			want: `"☃"="'\" \\ / \b \f \n \r \t \x00"`,
```

Comment:
Please add another unit test to verify that the "&" behavior works as intended:

ex:

```
in: []KeyValue{
  {"MY_URL":"'https://abc123.com/q?key1=value1&key2=value2'"}
},
want: `"MY_URL"="'https://abc123.com/q?key1=value1&key2=value2'",
```

Reply:
Done, added test case for html symbols